src/main/java/fr/foodlimit/api/food/FoodController.java

```diff
@@ -58,8 +80,14 @@ public void deleteFood(@PathVariable("id") Long id) {
    * @return
    */
   @PostMapping
-  public Food createFood(@PathVariable("placeId") Long placeId, @RequestBody Food food) {
-    return foodService.createFood(food, placeId);
+  public ResponseEntity<Food> createFood(HttpServletRequest request, @PathVariable("placeId") Long placeId, @RequestBody Food food) {
+    Place place = placeService.getPlace(placeId);
+
+    if (!this.environment.getActiveProfiles()[0].equals("test")) {
+      if (!checkPlace(request, place)) return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)